VideoLocker/res/values/strings.xml

```diff
@@ -401,7 +408,12 @@
     <string name="creating_account_text">Creating account...</string>
     <!-- Registration screen title -->
     <string name="register_title">Sign up for edX</string>
-
+    <!-- Text for successful registration with social token  -->
+    <string name="sign_up_with_facebook_ok">You\'ve successfully signed in with Facebook .</string>
+    <string name="sign_up_with_google_ok">You\'ve successfully signed in with Google .</string>
```

Comment:
When signed in , do we need to show the message ? We should just take the user to the "My Courses" screen.